### PR TITLE
Begin implementation of hosting static files.

### PR DESCRIPTION
**Pull request by joeldenning**
After this code, going to http://127.0.0.1:8787/walmart/apps/navbar/c1a777c770ee187cebedd0724653c771495f2af9/react-mf-navbar.js in the browser shows a javascript file that comes from https://react.microfrontends.app/navbar/c1a777c770ee187cebedd0724653c771495f2af9/react-mf-navbar.js

### CODE REVIEW

File: src/handleApps.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+} from "./getOrgSettings";
+import { notFoundResponse } from "./responseUtils";
+
+export async function handleApps(
+  request: Request,
+  params: Params
+): Promise<Response> {
+  const orgSettings = await getOrgSettings(params.orgKey);
+
+  if (!orgSettings.orgExists) {
```
Comment (Qirhi) [inline src/handleApps.ts:15]: Are we guaranteed to have a non-null orgSettings here?

**Review by Qirhi — APPROVED** [review]
Good job! Just one comment in handleApps.test related to possible edge case.  The logic was very clear and easy to read.  However, I had to look up the TypeScript recursive partial.
